src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IInterpolatedStringExpression.cs

```diff
@@ -444,5 +444,1026 @@ public void M(int x)
 
             VerifyOperationTreeAndDiagnosticsForTest<InterpolatedStringExpressionSyntax>(source, expectedOperationTree, expectedDiagnostics);
         }
+
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
+        [Fact]
+        public void InterpolatedStringExpression_Empty_Flow()
+        {
+            string source = @"
+using System;
+
+internal class Class
+{
+    public void M()
+    /*<bind>*/{
+        Console.WriteLine($"""");
```

Comment:
>Console.WriteLine( [](start = 8, length = 18)

Please keep tests scenarios to absolutely required minimum. Please remove invocations.  We usually use an assignments to a parameter, this creates very small amount of "noise" in the base-line

Reply:
Sure, I will keep in mind of next PRs.

---
In reply to: [185620391](https://github.com/dotnet/roslyn/pull/26545#discussion_r185620391) [](ancestors = 185620391)

Reply:
> Sure, I will keep in mind for next PRs.

Could you please make this change in this PR? I haven't looked at the tests yet. And probably no one have. This will simplify the review.

Reply:
The feedback is well taken, and I would ensure this for the next PRs. However, I would like to push back on rewriting the unit tests for the current PR just to reduce the test baseline by 3-4 lines, especially as there will be no difference in basic block/branch structure between the current approach and the recommended approach. Tagging @dotnet/analyzer-ioperation @dotnet/roslyn-compiler to get more opinions on whether this is significant hindrance for reviewers to warrant a rewrite.